src/main/java/uk/nhs/digital/nhsconnect/nhais/model/edifact/PersonName.java

```diff
@@ -85,37 +88,31 @@ public String getValue() {
         List<String> values = new ArrayList<>();
         values.add(QUALIFIER);
 
-        String namesDelimiter = containsName() ? "++" : "";
-        Optional.ofNullable(this.nhsNumber)
-            .map(value -> value + ":" + this.patientIdentificationType.getCode() + namesDelimiter)
-            .ifPresentOrElse(values::add, () -> values.add(namesDelimiter));
-
-        Optional.ofNullable(this.familyName)
+        values.add(Optional.ofNullable(this.nhsNumber)
+            .map(value -> value + ":" + this.patientIdentificationType.getCode())
+            .orElse(StringUtils.EMPTY));
+        values.addAll(IntStream.range(0, 2)
+            .mapToObj(x -> StringUtils.EMPTY)
+            .collect(Collectors.toList()));
```

Comment:
Wouldn't just `values.addAll(List.of(StringUtils.EMPTY, StringUtils.EMPTY))` or
````java
values.add(StringUtils.EMPTY);
values.add(StringUtils.EMPTY)
````
 be easier to read ? It might even be extracted to method `addSeparator(values)`